addons/point_of_sale/static/src/app/store/models.js

```diff
@@ -1658,7 +1658,7 @@ export class Order extends PosModel {
             footer: this.pos.config.receipt_footer,
             // FIXME: isn't there a better way to handle this date?
             shippingDate:
-                this.shippingDate && formatDate(DateTime.fromJSDate(new Date(this.shippingDate))),
+                this.shippingDate && formatDate(DateTime.fromISO(this.shippingDate)),
```

Comment:
small detail: If the format is "yyyy-mm-dd" or other sql format, I think the normal way is to use `DateTime.fromSQL` but it doesn't seem to matter if there is only the date part.

Reply:
I have changed it to be `fromSQL`, in this case, they both seem to function the same way.